generator/internal/dart/dart.go

```diff
@@ -174,9 +185,23 @@ func enumValueName(e *api.EnumValue) string {
 	return strcase.ToLowerCamel(e.Name)
 }
 
-func httpPathFmt(_ *api.PathInfo) string {
+func httpPathFmt(pathInfo *api.PathInfo) string {
```

Comment:
I think this function deserves some unit tests now.

Reply:
Done!